Modules/Projects/Core/Script_MiscUtilities.lua

```diff
@@ -44,6 +44,27 @@ function LogGsub(key, vars, color)
     LogRaw(strval, color)
 end
 
+function PreySense(pred, style, does_digest)
+    if(Player.PreySense)then
+    local color
+    local vtype 
+        if style == 0 then vtype = "OV" 
+        elseif style == 1 then vtype = "AV" 
+        elseif style == 2 then vtype = "CV" 
+        elseif style == 3 then vtype = "UB" 
+        end
+    local vfinal
+        if fatal then 
+            vfinal = "Fatal"
+            color = Color.Red
+        else 
+            vfinal = "Endo" 
+            color = Color.Neutral
+        end
+    LogRaw("You have that feeling in your gut...  "..pred.Name..":"..vtype..", "..vfinal.."", color)
```

Comment:
Pedantic: please ensure there are spaces around all `..` operators. Also, I think you're missing a space after the `:`.

Reply:
This was not yet addressed.